### DISCUSSION TITLE
Is there a way of adding background opacity to a component?

### DISCUSSION THREAD

**Discussion by ChristineTham** [chakra-ui/chakra-ui#2996] [Q&A] [answered]
Is there a way of adding opacity to the background color component without affecting child elements?

Something like this

```
<Box bgOpacity="25%" bg="black" bgImage="url('/some/image')">
  <Text>I am not faded!</Text>
</Box>
```

I have tried setting `bg="rgba(0,0,0,0.25)"`  but that doesn't work, unlike in CSS.

**Comment by chasinhues** [comment, marked as answer]
I checked out this article and made a sandbox as an example of how to achieve an opaque background color/image that doesn't interfere with child text: https://coder-coder.com/background-image-opacity/

## Demo:
https://codesandbox.io/s/background-image-opacity-demo-e6ihx

**Reply by ChristineTham** [reply]
> ## Demo:
> https://codesandbox.io/s/background-image-opacity-demo-e6ihx

Thank you so very much! I managed to get it working and credited you!

[https://my-chakra-components.now.sh/headers](https://my-chakra-components.now.sh/headers)

**Reply by chasinhues** [reply]
Glad to hear you got it working, website looks great - nice work! 🎉

**Comment by unurag** [comment]
Well there is a easy work around 😄 

bgImage="linear-gradient(rgba(0, 0, 0, 0.527),rgba(0, 0, 0, 0.5)) , url('Background.jpg')"

**Reply by astroboy28** [reply]
Thanks unurag...
